tensorzero-core/src/optimization/gepa.rs

```diff
@@ -0,0 +1,350 @@
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[cfg(feature = "pyo3")]
+use pyo3::prelude::*;
+
+use crate::error::Error;
+use crate::model_table::ProviderTypeDefaultCredentials;
+use crate::utils::retries::RetryConfig;
+use crate::variant::chat_completion::UninitializedChatCompletionConfig;
+
+// Default functions
+fn default_batch_size() -> usize {
+    5
+}
+
+fn default_max_iterations() -> u32 {
+    1
+}
+
+fn default_max_concurrency() -> u32 {
+    10
+}
+
+fn default_timeout() -> u64 {
+    300
+}
+
+fn default_analysis_model() -> String {
+    "anthropic::claude-sonnet-4-5-20250929".to_string()
+}
+
+fn default_mutation_model() -> String {
+    "anthropic::claude-sonnet-4-5-20250929".to_string()
+}
```

Comment:
This is going to rot very quickly and it's unclear if they'll have one. I don't think we should have a default. Mandatory to provide.